src/main/java/com/be3c/sysmetic/domain/member/controller/MemberManagementController.java

```diff
@@ -0,0 +1,128 @@
+public class MemberManagementController {
+
+    private final MemberManagementService memberManagementService;
+
+    /*
+        관리자 회원 조회 api
+        1. 회원 조회 성공했을 때 : OK
+        2. 회원 조회 실패했을 때 : INTERNAL_SERVER_ERROR
+        2. SecurityContext에 userId가 존재하지 않을 떄 : FORBIDDEN
+    */
+    @Operation(
+            summary = "관리자 회원 조회",
+            description = "관리자가 특정 역할(role)과 검색 조건을 기반으로 회원 목록을 조회하는 API"
+    )
+    @ApiResponses({
+            @ApiResponse(
+                    responseCode = "200",
+                    description = "회원 조회 성공",
+                    content = @Content(mediaType = "application/json")
+            ),
+            @ApiResponse(
+                    responseCode = "403",
+                    description = "SecurityContext에 userId가 존재하지 않음 (FORBIDDEN)",
+                    content = @Content(mediaType = "application/json")
+            ),
+            @ApiResponse(
+                    responseCode = "500",
+                    description = "서버 오류로 인해 회원 조회 실패",
+                    content = @Content(mediaType = "application/json")
+            )
+    })
+    @GetMapping("/admin/members")
+    public ResponseEntity<APIResponse<PageResponse<MemberGetResponseDto>>> getMemberPage(@RequestParam String role, @RequestParam(defaultValue = "1") Integer page, @RequestParam(defaultValue = "all", required = false) String searchType, @RequestParam(required = false) String searchKeyword) {
+        /*
+            조회 대상) all, user, trader, manager
+            검색 필터) all, email, name, nickname, phoneNumber, birth
+        */
+        PageResponse<MemberGetResponseDto> members = memberManagementService.findMemberPage(role, page, searchType, searchKeyword);
+        return ResponseEntity.status(HttpStatus.OK).body(APIResponse.success(members));
+    }
+
+    /*
+        관리자 회원 등급변경 api
+        1. 등급변경 성공했을 때 : OK
+        2. 등급변경 실패했을 때 : INTERNAL_SERVER_ERROR
+        2. SecurityContext에 userId가 존재하지 않을 떄 : FORBIDDEN
+    */
+    @Operation(
+            summary = "관리자 회원 등급 변경",
+            description = "관리자가 특정 회원의 등급을 변경하는 API"
+    )
+    @ApiResponses({
+            @ApiResponse(
+                    responseCode = "200",
+                    description = "회원 등급 변경 성공",
+                    content = @Content(mediaType = "application/json")
+            ),
+            @ApiResponse(
+                    responseCode = "403",
+                    description = "SecurityContext에 userId가 존재하지 않음 (FORBIDDEN)",
+                    content = @Content(mediaType = "application/json")
+            ),
+            @ApiResponse(
+                    responseCode = "500",
+                    description = "서버 오류로 인해 등급 변경 실패",
+                    content = @Content(mediaType = "application/json")
+            )
+    })
+    @PatchMapping("/admin/members")
+    public ResponseEntity<APIResponse<String>> changeMemberRoleCode(@RequestParam Long memberId, @RequestParam boolean changeRoleCode) {
```

Comment:
```suggestion
    @PatchMapping("/admin/members/{memberId}")
    public ResponseEntity<APIResponse<String>> changeMemberRoleCode(@PathValiable Long memberId, @RequestParam boolean changeRoleCode) {
```
위와 같이 변경하면 더 좋을 거 같습니다

아래 내용 참고해보세요
[When to use @QueryParam vs @PathParam](https://stackoverflow.com/questions/11552248/when-to-use-queryparam-vs-pathparam)
[REST API URI Naming Conventions and Best Practices](https://restfulapi.net/resource-naming/)